core/templates/pages/exploration-player-page/services/audio-preloader.service.ts

```diff
@@ -93,15 +100,24 @@ export class AudioPreloaderService {
   private getAudioFilenamesInBfsOrder(sourceStateName: string): string[] {
     const languageCode = (
       this.audioTranslationLanguageService.getCurrentAudioLanguageCode());
-    const allVoiceovers = this.exploration.getAllVoiceovers(languageCode);
-    const bfsTraversalOfStates = (
-      this.computeGraphService.computeBfsTraversalOfStates(
-        this.exploration.getInitialState().name, this.exploration.getStates(),
-        sourceStateName));
+    let allVoiceovers = null;
+    if (languageCode !== null) {
```

Comment:
When can `languageCode` be `null`?

Reply:
cover in PR [https://github.com/oppia/oppia/pull/15027](url), take a pull from that.

Reply:
Okay.